diff --git a/CMakeLists.txt b/CMakeLists.txt
index fbebe99e7..ff83a495a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -262,7 +262,8 @@ include_directories(SYSTEM ${OPENSCENEGRAPH_INCLUDE_DIRS})
 set(USED_OSG_PLUGINS
                     osgdb_bmp
                     osgdb_dds
-                    osgdb_freetype
+                    # osgdb_freetype
+                    # leave this out for now
                     osgdb_jpeg
                     osgdb_osg
                     osgdb_png
